package.json

```diff
@@ -22,7 +22,9 @@
     "redux": "^3.7.2",
     "redux-immutable": "^4.0.0",
     "redux-thunk": "^2.2.0",
-    "reqwest": "^2.0.5"
+    "reqwest": "^2.0.5",
```

Comment:
Will this library `reqwest` work for image upload as well ? Or do we need to move to `axios` ?

Reply:
We are using axios only. Forgot to remove `reqwest` package. Will remove it.